tests/unit/common/test_table.py

```diff
@@ -996,8 +997,8 @@ def test_add_opt_column(self):
         table.add_column(name='col8', description='column #8', index=True, table=True)
         self.assertEqual(table.col8.description, 'column #8')
 
-        table.add_column(name='col10', description='column #10', index=True, col_cls=VocabData)
-        self.assertIsInstance(table.col10, VocabData)
+        table.add_column(name='col11', description='column #11', enum=True, index=True)
+        self.assertIsInstance(table.col11, EnumData)
```

Comment:
Can we keep the previous code regarding col10 to test add_column with col_cls=EnumData?

```suggestion
        table.add_column(name='col10', description='column #10', index=True, col_cls=EnumData)
        self.assertIsInstance(table.col10, EnumData)

        table.add_column(name='col11', description='column #11', enum=True, index=True)
        self.assertIsInstance(table.col11, EnumData)
```